prisma-integration.mdx

```diff
@@ -0,0 +1,218 @@
+After generating your schema. You’ll need to run `npx prisma generate` to get typesafety in your application code.
+
+## Running queries
+
+Create an `index.ts` file at the project root with the following:
+
+```jsx
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+async function main() {
+  const record = await prisma.prices_med_2500.findMany({
+    where: {
+      symbol: {
+        equals: 'PATT'
+      }
+    }
+  });
+  console.log('record', record);
+}
+
+main()
+  .then(async () => {
+    await prisma.$disconnect();
+  })
+  .catch(async (e) => {
+    console.error(e);
+    await prisma.$disconnect();
+    process.exit(1);
+  });
+```
+
+### Driver Adapters (Optional)
+
+Client initialisation for driver adapters is slightly different:
+
+```jsx
+import { Pool } from 'pg';
+import { PrismaPg } from '@prisma/adapter-pg';
+import { PrismaClient } from '@prisma/client';
+
+const connectionString = `${process.env.DATABASE_URL}`;
+
+const pool = new Pool({ connectionString });
+const adapter = new PrismaPg(pool);
+const prisma = new PrismaClient({ adapter });
+async function main() {
+  const record = await prisma.prices_med_2500.findMany({
+    where: {
+      symbol: {
+        equals: 'PATT'
+      }
+    }
+  });
+  console.log('record', record);
+}
+
+main()
+  .then(async () => {
+    await prisma.$disconnect();
+  })
+  .catch(async (e) => {
+    console.error(e);
+    await prisma.$disconnect();
+    process.exit(1);
+  });
+```
+
+## Changing the schema
+
+You may want to add, delete, or modify your tables and your fields. Changes can come from the Prisma Schema file and be `pushed` onto the remote database, or they can come from the remote database and be `pulled` into the Prisma Schema file which would update your Prisma Client types. Here is an example of how you could add a new field to your existing database schema via the Prisma Schema file and have those changes applied to your remote database.
+
+```jsx
+model companies_med_250 {
+  city            String? // This field is new.
+  address         String?
+  catch_phrase    String?
+  ceo             String?
+  phone           String?
+  email           String?
+  exchange        String?
+  xata_updatedat  DateTime          @default(now()) @db.Timestamptz(6)
+  xata_id         String            @unique(map: "_pgroll_new_companies_med_250_xata_id_key") @default(dbgenerated("('rec_'::text || (xata_private.xid())::text)"))
+  xata_version    Int               @default(0)
+  xata_createdat  DateTime          @default(now()) @db.Timestamptz(6)
+  prices_med_2500 prices_med_2500[]
+}
+```
+
+After saving this file, run `prisma db push` . The column should be added to your remote database and your Prisma Client should be aware of the new field available on your model.
+
+## Get access
+
+This functionality is only available in our public beta Xata 2.0 version as it relies on our new wire protocol. Sign up for the public beta [here](). Happy coding!
```

Comment:
@alexfrancoeur  you'll probably want to add some information to this call to action section and then maybe add a link to the page where people can sign up for public beta.